truefoundry.yaml

```diff
@@ -30,12 +29,12 @@ components:
         build_context_path: ./
       # Change this later
       build_source:
-        ref: fdfce2e13d5939d7d1c1fce57b90748d8cadd65b
-        type: git
-        repo_url: https://github.com/truefoundry/cognita
-        branch_name: ps_cleanup_metadatastore
-        # local_build: false
-        # type: local
+        # ref: fdfce2e13d5939d7d1c1fce57b90748d8cadd65b
+        # type: git
+        # repo_url: https://github.com/truefoundry/cognita
+        # branch_name: ps_cleanup_metadatastore
+        local_build: false
+        type: local
```

Comment:
Why are we commenting / uncommenting different configurations?
If needed, let's keep it local build only

Github workflow can be adjusted to replace entire build_source

Reply:
Sure, I'll keep it to local, we can modify the github workflow

Reply:
Please add a ticket for this

Reply:
Done.